payments-core/res/values-DA/strings.xml

```diff
@@ -76,14 +76,6 @@
   <string name="address_zip_invalid">ZIP-koden er ugyldig.</string>
   <!-- Error text indicating zip/ postal code is invalid, used for international addresses -->
   <string name="address_zip_postal_invalid">ZIP-kode/Postnr. er ugyldigt</string>
-  <!-- Pay over time with Affirm copy -->
-  <string name="affirm_buy_now_pay_later">Betal over tid med &lt;img src=\"affirm\"&gt;.</string>
-  <!-- Label for Name on account field on form -->
-  <string name="au_becs_account_name">Navn på konto</string>
-  <!-- Placeholder for AU BECS BSB number -->
-  <string name="au_becs_bsb_number">BSB-nummer</string>
-  <!-- Legal text shown when using AUBECS. -->
-  <string name="au_becs_mandate">Ved at angive dine bankoplysninger og bekræfte denne betaling accepterer du denne direkte debiteringsanmodning samt &lt;a href=\\"https://stripe.com/au-becs-dd-service-agreement/legal\\"&gt;debiteringsanmodningens serviceaftale&lt;/a&gt;, og autoriserer Stripe Payments Australia Pty Ltd ACN 160 180 343, direkte debiterings-bruger-ID 507156 (“Stripe”) til at debitere din konto via Bulk Electronic Clearing System (BECS) på vegne af %s (\"Forhandleren\") for eventuelle beløb, der separat meddeles dig af Forhandleren. Du bekræfter, at du enten er kontoindehaver eller autoriseret underskriver på den konto, der er anført ovenfor.</string>
```

Comment:
Do you know why this was removed?

Reply:
I am not sure, all I did was run the lokalise script. I think the translations were removed on lokalise

Reply:
Can you revert these changes that are unrelated to yours and then we'll fix this in Lokalise?

Reply:
Created RUN_MOBILESDK-948